generators/addons/index.js

```diff
@@ -96,7 +96,7 @@ module.exports = class extends Generator {
 
             // Append Azure DevOps options if selected
             requestedLibraries = this.options.ci === undefined ? requestedLibraries :
-                requestedLibraries.concat('continuousIntegrationKarma');
+                requestedLibraries.concat('continuousIntegrationJest');
```

Comment:
@StfBauer this should probably also be conditionnal to the user selecting jest as well.

Reply:
@baywet What do you think from a UX perspective. We first ask for Azure DevOps option make it visible saying "Azure DevOps incl. Jest". Then we don't have to ask for Jest anymore.

Only if you selected you don't want to have "Azure DevOps" then ask the "Jest Testing Framework".

Reply:
I depends on where things go, if we have a few unit testing options and a few CI technologies, this approach might be ok. However if you have 3 unit testing frameworks and 3 CI options, you have to maintain 9 choices, there is no separation of concern, which makes maintenance harder etc...
I'd like both to be as separated as possible, I'm going to check if I can simply detect that jest was selected, without changing what users see

Reply:
I'm able to read the global answers object and get the testing framework, so we're good on that side